core/lib/resources/presence/sentry.js

```diff
@@ -1,159 +1,264 @@
-var logging = require('minilog')('radar:presence:sentry'),
-    Persistence = require('persistence');
+var _ = require('underscore'),
+    Minilog = require('minilog'),
+    logging = Minilog('radar:sentry'),
+    Persistence = require('persistence'),
+    redisSentriesKey = 'sentry:/radar';
 
-function parse(message) {
-  try {
-    return JSON.parse(message);
+var defaultOptions = {
+  DEFAULT_EXPIRY_OFFSET: 4000,
+  REFRESH_INTERVAL: 3500,
+  CHECK_INTERVAL: 10000
+};
```

Comment:
these are too low for even defaults. I think it should be 60 seconds, 10 seconds, 30 seconds.